packages/shared/src/features/onboarding/steps/FunnelPricing.tsx

```diff
@@ -142,15 +141,23 @@ export const FunnelPricing = ({
     cta,
   };
 
+  useEffect(() => {
```

Comment:
Let's add a todo here to support resuming funnel, so i a user resumes a funnel we should use the time the step was created originally